apps/docs/src/pages/components/form-element.mdx

```diff
@@ -90,19 +90,22 @@ along with the following building blocks provided by UI Kit:
 
 ```tsx live
 <HvFormElement required status="invalid">
-  <HvLabel label="Name">
-    <HvInfoMessage>Input your name</HvInfoMessage>
-    <HvBaseInput
-      placeholder="Insert name"
-      endAdornment={
-        <>
-          <HvAdornment showWhen="invalid" icon={<Fail color="negative" />} />
-          <HvAdornment showWhen="valid" icon={<Success color="positive" />} />
-        </>
-      }
-    />
-  </HvLabel>
-  <HvWarningText disableBorder>This is the error message.</HvWarningText>
+  <div className="flex gap-xs">
```

Comment:
`HvFormElement` accepts in a `className` - no need for another `div`!

Reply:
D'oh! fixed it, spot the `div`erence.